angular-action-sheet.js

```diff
@@ -30,6 +30,12 @@ function jbActionSheetDirective($document) {
   }
 
   function controller ($scope) {
+    $document.bind('keydown', function(e) {
```

Comment:
Does angular automatically unbind this somehow or will this handler continue to fire even after the sheet is closed?

Also, a short comment indicating the key (esc) being bound would be helpful.